api/src/pcapi/core/offers/api.py

```diff
@@ -593,7 +594,10 @@ def create_stock(
                 stock=created_stock,
             )
         )
-    repository.add_to_session(created_stock, *created_activation_codes)
+    if offer.lastValidationPrice is None and offer == offer_mixin.OfferValidationStatus.APPROVED:
```

Comment:
Une erreur d'inattention dans la précipitation avant la MES ?
```suggestion
    if offer.lastValidationPrice is None and offer.validation == offer_mixin.OfferValidationStatus.APPROVED:
```
ou mieux :
```suggestion
    if offer.lastValidationPrice is None and offer.isApproved:
```